includes/blocks.inc

```diff
@@ -623,7 +623,6 @@ function islandora_solr_advanced_search_form($form, &$form_state) {
     $term['field'] = array(
       '#title' => t('Field'),
       '#type' => 'select',
-      '#default_value' => isset($value['field']) ? $value['field'] : 'dc.title',
```

Comment:
Discussed in Committers', but just to have it in writing:

I believe the `isset($value['field']) ? $value['field']` bit is necessary to support selecting the appropriate fields when the "advanced search" block is present on your Solr results page.

... Instead of wiping out the `#default_value` entirely, it may suffice to use `NULL` instead of `'dc.title'`... alternatively, we could probably add in the `#default_value` immediately after this field if `$value['field']` is set.

Reply:
Tested it out - you're right: without a "default" line, if the advanced search block is in the search results, it shows the first field in the list and your current search term, instead of the actual field you selected.

Changed it to Null and it works as desired!